lucene/analysis/common/src/java/org/apache/lucene/analysis/miscellaneous/TypeAsSynonymFilterFactory.java

```diff
@@ -46,10 +49,17 @@
   public static final String NAME = "typeAsSynonym";
 
   private final String prefix;
+  private Set<String> ignore = null;
+  private final int synFlagMask;
 
   public TypeAsSynonymFilterFactory(Map<String,String> args) {
     super(args);
     prefix = get(args, "prefix");  // default value is null
+    String ignoreList = get(args, "ignore");
+    synFlagMask = getInt(args,"synFlagsMask", ~0);
+    if (ignoreList != null) {
+      ignore = Set.of(ignoreList.split(","));
+    }
```

Comment:
There is a method `getSet()` which you could use instead of `get()` for the ignoreList; it does the splitting for you, and also supports mixed space/comma separators.

Reply:
Ah cool.